ibis/expr/types/temporal.py

```diff
@@ -253,6 +253,67 @@ def __rsub__(self, other: ops.Value[dt.Interval | dt.Time, ds.Any]):
 
     rsub = __rsub__
 
+    def delta(
+        self, other: datetime.time | Value[dt.Time], part: str
+    ) -> ir.IntegerValue:
+        """Compute the number of `part`s between two times.
+
+        ::: {.callout-note}
+        ## The order of operands matches standard subtraction
+
+        The second argument is subtracted from the first.
+        :::
+
+        Parameters
+        ----------
+        other
+            A time expression
+        part
+            The unit of time to compute the difference in
+
+        Returns
+        -------
+        IntegerValue
+            The number of `part`s between `self` and `other`
+
+        Examples
+        --------
+        >>> import ibis
+        >>> ibis.options.interactive = True
+        >>> start = ibis.time("01:58:00")
+        >>> end = ibis.time("23:59:59")
+        >>> end.delta(start, "hour")
+        22
+        >>> data = '''tpep_pickup_datetime,tpep_dropoff_datetime
+        ... 2016-02-01T00:23:56,2016-02-01T00:42:28
+        ... 2016-02-01T00:12:14,2016-02-01T00:21:41
+        ... 2016-02-01T00:43:24,2016-02-01T00:46:14
+        ... 2016-02-01T00:55:11,2016-02-01T01:24:34
+        ... 2016-02-01T00:11:13,2016-02-01T00:16:59'''
+        >>> with open("/tmp/triptimes.csv", "w") as f:
+        ...     _ = f.write(data)
```

Comment:
Apparently I never hit enter...

Anyway, my question:

I wonder if we should endeavor to avoid using underscores in docstrings that aren't specifically `Deferred`?

Reply:
Yeah, I think that's a good idea. The problem I was having is that since `f.write` returns a value the doctest expects some output.

Is there a better way to handle ignoring this output?

Reply:
I suppose I could simply assign it to a different variable that I never use.

Reply:
Oh, it looks like I can simply use `doctest: +SKIP`, sweet.

Reply:
I will clean these up across the codebase in a follow-up. Thanks for the callout!